boards/samd21-xpro/include/periph_conf.h

```diff
@@ -242,7 +242,8 @@ static const spi_conf_t spi_config[] = {
         .mosi_mux = GPIO_MUX_D,
         .clk_mux  = GPIO_MUX_D,
         .miso_pad = SPI_PAD_MISO_0,
-        .mosi_pad = SPI_PAD_MOSI_2_SCK_3
+        .mosi_pad = SPI_PAD_MOSI_2_SCK_3,
+        .gclk_src = 0
```

Comment:
Here you see how the lack of a comma makes the diff bigger than it should be

Reply:
BTW. This was me.